assets/elm/Data/Board.elm

```diff
@@ -128,8 +155,13 @@ nextSnakeAndApples time snake apples =
 
 
 score : Board -> Int
-score { snake } =
-    List.length snake.body
+score { currentPlayerId, snakes } =
+    case Dict.get currentPlayerId snakes of
+        Nothing ->
+            0
```

Comment:
... I suppose it's just a naming problem, though... if `maybe` were actually called `maybeWithDefaultMapped` it might be fine.